code/modules/mob/login.dm

```diff
@@ -5,15 +5,18 @@
 	computer_id	= client.computer_id
 	log_access("Login: [key_name(src)] from [lastKnownIP ? lastKnownIP : "localhost"]-[computer_id] || BYOND v[client.byond_version].[client.byond_build]")
 	for(var/mob/M in GLOB.player_list)
-		if(M == src)	continue
-		if( M.key && (M.key != key) )
+		if(M == src)	
+			continue
+		if(M.ckey && (M.ckey != ckey) )
 			var/matches
-			if( (M.lastKnownIP == client.address) )
+			if((M.lastKnownIP == client.address) )
 				matches += "IP ([client.address])"
-			if( (client.connection != "web") && (M.computer_id == client.computer_id) )
-				if(matches)	matches += " and "
+			if((client.connection != "web") && (M.computer_id == client.computer_id))
+				if(matches)	
+					matches += " and "
 				matches += "ID ([client.computer_id])"
-				spawn() alert("You have logged in already with another key this round, please log out of this one NOW or risk being banned!")
+				spawn(1) 
```

Comment:
Evil CALLBACK whispers...

Reply:
In client login procs and similar you cannot use callbacks as it would often break things. This was done on purpose.